services/app-api/handlers/users/create.js

```diff
@@ -8,43 +9,66 @@ export const main = handler(async (event, context) => {
     return null;
   }
 
-  console.log("zzzEvent", event);
   const data = JSON.parse(event.body);
-  // const data = event.body;
-  // return data;
+
   console.log(JSON.stringify(event, null, 2));
```

Comment:
please remove console logs

Reply:
This is a serverside console log that is part of the framework and used for long-term logging reasons. I'd prefer not to remove it.

Reply:
yea it cool in local, but we dont want console logs in prod. once you tested the code fully all console logs should be removed. it is not part of the framework, someone put that there for testing purpose.